docs/flow/modin/experimental/xgboost.rst

```diff
@@ -0,0 +1,132 @@
+Modin XGBoost module description
+""""""""""""""""""""""""""""""""
+High-level Module Overview
+''''''''''''''''''''''''''
+
+This module holds classes, public interface functions and internal functions for making distributed 
+xgboost in Modin.
+
+Public classes :ref:`Booster <Booster>`, :ref:`DMatrix <DMatrix>` and function  :ref:`train <train>` provide
+to user familiar interfaces of xgboost. That functionality is placed in ``modin.experimental.xgboost.xgboost`` module.
+
+Internal module ``modin.experimental.xgboost.xgboost.xgboost_ray`` holds implementation of Modin XGBoost
+for Ray backend. This module mainly consist of Ray actor-class :ref:`ModinXGBoostActor <Actor>`,
+function to distribution Modin's partitions between actors :ref:`_assign_row_partitions_to_actors <assign_parts>`,
+internal :ref:`_train <internal_train>`/:ref:`_predict <internal_predict>` functions used from the public interfaces 
+and additional util functions for computing cluster resources, actor creations etc.
+
+Public interfaces
+'''''''''''''''''
+
+Class :ref:`DMatrix <DMatrix>` inherits original class ``xgboost.DMatrix`` and overrides constructor which supports
+only `data` and `label` parameters now. Both of the parameters must be ``modin.pandas.DataFrame``,
+those dataframes will be unwrapped to lists of delayed objects of Modin's row partitions using function
+``modin.distributed.dataframe.pandas.unwrap_partitions``.
```

Comment:
```suggestion
:func:`~modin.distributed.dataframe.pandas.unwrap_partitions`.
```
assuming we already have that function documented this should generate a cross-link to it.

Reply:
Suggestion was applied, but cross-link on `unwrap_partitions` (this function is documented [here](https://github.com/modin-project/modin/blob/master/docs/developer/pandas/partition_api.rst)) wasn't created.

Reply:
hmm, this is something to look at when there's some spare time I guess